shared/js/media/media_frame.js

```diff
@@ -107,7 +108,8 @@ MediaFrame.prototype.displayImage = function displayImage(blob,
                                                           height,
                                                           preview,
                                                           rotation,
-                                                          mirrored)
+                                                          mirrored,
+                                                          timestamp)
```

Comment:
If the blob is a file, you can get the timestamp from it. If the blob is not a file, then we won't have a meaningful timestamp anyway.